test/fail_compilation/fail15896.d

```diff
@@ -0,0 +1,18 @@
+// PERMUTE_ARGS:
+// REQUIRED_ARGS: -de
+/*
+TEST_OUTPUT:
+---
+fail_compilation/fail15896.d(11): Deprecation: module imports.imp15896 member `thebar` is not accessible
+fail_compilation/fail15896.d(11): Deprecation: module imports.imp15896 member `packagebar` is not accessible
```

Comment:
Looks like you need to highlight the module name now.

Reply:
done.